src/mount/linux.rs

```diff
@@ -85,7 +88,7 @@ libc_bitflags!(
         MNT_DETACH;
         /// Mark the mount point as expired.
         MNT_EXPIRE;
-        /// Don't dereference `target` if it is a symlink.  
```

Comment:
Again: I believe you have your editor set to automatically delete trailing whitespaces. I'm not against it, but that should be done as a separate clean up commit.